test/e2e/thanos_querier_controller_test.go

```diff
@@ -139,31 +137,28 @@ func newThanosQuerier(t *testing.T, name string, selector map[string]string) *ms
 }
 
 func waitForThanosQuerierDeletion(tq *msov1.ThanosQuerier) error {
-	//nolint
-	return wait.Poll(5*time.Second, wait.ForeverTestTimeout, func() (bool, error) {
-		err := f.K8sClient.Get(context.Background(),
+	return wait.PollUntilContextTimeout(context.Background(), 5*time.Second, wait.ForeverTestTimeout, true, func(ctx context.Context) (done bool, err error) {
```

Comment:
General question: I see that we replace some function signatures with named parameters but not all of them, is there a concrete motivation? 
Personally I'm not a big fan of named parameters but I think that we need to be consistent.

Reply:
wait.Poll is deprecated in this version, tried to change for PollUntilContextTimeout all over the code but in a couple of funcs there are issues with context, then I let wait.Poll with //nolint and I'll try to fix in next iteration.

Reply:
in library examples PollUntilContextTimeout is done in this way, I'll change to named parameters in all funcs. thanks